server/datastore/mysql/apple_mdm.go

```diff
@@ -1655,9 +1686,19 @@ func (ds *Datastore) IngestMDMAppleDeviceFromOTAEnrollment(
 				HardwareSerial: deviceInfo.Serial,
 				PlatformHint:   deviceInfo.Product,
 				HardwareModel:  deviceInfo.Product,
+				UUID:           &deviceInfo.UDID,
```

Comment:
@gillespi314 Also could be nice with your input here, if it is okay to add the UUID already here, instead of waiting for the first MDM check-in command.